web/client/components/map/openlayers/__tests__/Map-test.jsx

```diff
@@ -168,7 +168,7 @@ describe('OpenlayersMap', () => {
 
         const olMap = map.map;
         expect(olMap.getView().getZoom()).toBe(12);
-        map.setProps({zoom: 12, center: {y: 44, x: 10}});
+        map.setProps({zoom: 12.4, center: {y: 44, x: 10}});
         expect(olMap.getView().getZoom()).toBe(12);
```

Comment:
Same as before: here the test should expect a different value than before the setProps()